### DISCUSSION TITLE
RESPUESTA ANTES DE DARLE AL BOTON CONTINUAR???

### DISCUSSION THREAD

**Discussion by mgip** [creagia/laravel-redsys#45] [Q&A]
Hola, cuando sale el formulario de redsys, se introduce la tarjeta y luego se le da a pagar, procesa el pago, luego se le tiene que dar continuar para que redsys devuelva la respuesta a la URL OK o KO, el problema que veo es que necesito la respuesta antes que se le de a continuar, eso ocurre porque muchos no le dan a continuar, muchos usuarios al ver la notificacion en verde ya lo dan como pagado, pero mi aplicacion no se entera de la respuesta, si se logra ver si fue OK o KO por canales, pero la idea es que el app reciba la informacion porque es un sistema de reservas. Lo estoy desarrollando en Laravel 9.

¿Es posible?, lo otro que estuve leyendo fue usar el API REST, pero me surge una duda, como gestiona los pagos que requieren la autorizacion del banco, como sms o gestionarla via app (del banco), porque via pasarale redsys como lo tengo ahorita, si sale el html del banco. 

De verdad estoy que me vuelvo loco, no encuentro informacion al respecto. 

Gracias de antemano por la ayuda.

**Comment by dtorras** [comment]
Hola. Cierro como duplicado para seguir la discussion en la original https://github.com/creagia/redsys-php/discussions/24
